### PR TITLE
Hide run action button when actions are disabled per database

### PR DESCRIPTION
**Pull request by kulyk**
Epic #27581

### Description

Hides the "Run action" button from the model detail page if actions are disabled for a database.

### How to verify

1. Go to `/admin/databases` and enable actions for your PSQL / MySQL database
2. Create a model using one of this database's table
3. Go to `/model/:modelId/detail/actions` and create a few actions
4. Ensure you can see a "play" icon next to each of your actions
5. Go back to `/admin/databases` and disable actions for that model's database
6. Go back to the model detail page, ensure the "play" buttons near actions are hidden now

### Demo

CleanShot 2023-02-21 at 18 56 29@2x

### Checklist

- [x] Tests have been added/updated to cover changes in this PR

- - -
This change is [Reviewable](https://reviewable.io/reviews/metabase/metabase/28501)

### CODE REVIEW

File: frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx [hunk 1]
```diff
@@ -117,6 +117,7 @@ function ModelActionListItem({
                   icon="play"
                   onlyIcon
                   tooltip={t`Run`}
+                  aria-label={t`Run`}
```
Comment (WiNloSt) [inline frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx:120]: This could be in another PR, but do you think it's worth changing the underlying component instead, so it will set `aria-label` to `tooltip` automatically?

But apparently, `tooltip` only applies when `as={Link}`, so I'm not totally sure how to properly make that change. But you get the idea.

**Review by WiNloSt — APPROVED** [review]
Looks good to me. No new string change this time 😅

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx (hunk 1 above)
Comment (kulyk) [reply frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx:120]: I'm not sure I'd generally expect a tooltip and an aria-label to match all the time. I'd keep them separate so there are no surprises when updating one of them

Comment (WiNloSt) [reply frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx:120]: You're right. Not all the time. This usually happens when the underlying elements having a tooltip doesn't have text inside themselves e.g. icons.
